src/Directory.Build.targets

```diff
@@ -54,4 +54,43 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
+
+  <PropertyGroup Condition="'$(IsInvokableTestProject)' == 'true'">
+    <LaunchSettingsOutputFileFullPath Condition="'$(LaunchSettingsOutputFileFullPath)' == ''">$(AppDesignerFolder)/launchSettings.json</LaunchSettingsOutputFileFullPath>
+    <LaunchSettingsInputFileFullPath Condition="'$(LaunchSettingsInputFileFullPath)' == ''">$(ToolsDir)tests.targets</LaunchSettingsInputFileFullPath>
```

Comment:
Why is `$(ToolsDir)tests.targets` the input file?  Shouldn't it be *this* file?

Reply:
The tests.targets file contains the properties that are written into the launchSettings.json file, therefore I use it as the input which means that the launchSettings files will be re-created when a new buildtools version comes in.

Reply:
Ah, got it.  makes sense.